controllers/tasks.js

```diff
@@ -16,6 +16,10 @@ const addNewTask = async (req, res) => {
     };
     const task = await tasks.updateTask(body);
 
+    if (task.userNotFound) {
+      logger.error("Error while creating new task: Incorrect username passed");
+      return res.boom.badRequest("Unable to add task");
```

Comment:
**Suggestion:** I think we should have a better message here maybe like user not found

Reply:
Thanks for the suggestion. The message is updated to `User not found`